src/components/connect.js

```diff
@@ -194,14 +322,14 @@ export default function connect(mapPropsToRequestsToProps, options = {}) {
         if (mapping.hasOwnProperty('value')) {
           return onFulfillment(meta)(mapping.value)
         } else {
-          const request = buildRequest(mapping)
+          const request = defaults.buildRequest(mapping)
           meta.request = request
           this.setAtomicState(prop, startedAt, mapping, initPS(meta))
 
-          const fetched = window.fetch(request)
+          const fetched = defaults.fetch(request)
```

Comment:
This line fails on “Uncaught TypeError: Illegal invocation” in Chrome. See http://stackoverflow.com/a/9678166/129492 for why. Changing it to `const fetched = defaults.fetch.call(window, request)` fixes it, but this probably need to be done in a way that holds onto the reference to `window` and probably more places than just this.